lib/onetime.rb

```diff
@@ -135,7 +136,7 @@ module Config
     attr_reader :env, :base, :bootstrap
     def load path=self.path
       raise ArgumentError, "Bad path (#{path})" unless File.readable?(path)
-      YAML.load_file path
+      YAML.load(ERB.new(File.read(path)).result)
```

Comment:
I see. This allows the config file to be interpolated. Could you add a comment with a usage example?